evm/src/cpu/kernel/asm/util/assertions.asm

```diff
@@ -34,10 +34,8 @@ global panic:
 %endmacro
 
 %macro assert_lt
-    // %assert_zero is cheaper than %assert_nonzero, so we will leverage the
-    // fact that (x < y) == !(x >= y).
-    GE
-    %assert_zero
+    LT
+    %assert_nonzero
```

Comment:
Can you please revert this, back with the original comment? Your change is incurring 1 additional CPU cycle per `assert_lt`

Reply:
The problem is that `GE` doesn't exists

Reply:
Ah! That's indeed unfortunate. The macro was never used? (in any case, disregard my comment then)